apps/www/content/docs/components/data-table.mdx

```diff
@@ -463,7 +463,7 @@ Let's make the email column sortable.
 
 ### Update `<DataTable>`
 
-```tsx showLineNumbers title="app/payments/data-table.tsx" showLineNumbers {3,6,10,18,25-28}
```

Comment:
This looks good. Can we do small PRs please? Would be great if you could break the other changes in different PRs. You can keep this one. We'll merge. Thank you.

Reply:
It is an honor to address you, thank you very much for reviewing my PR. I will roll back the changes from the `CONTRIBUTING.md` file to the latest version of `main` and make a different PR for it keeping only the `data-table.mdx` correction for this PR.